services/client/src/styles/_sidebar.scss

```
@@ -72,30 +82,35 @@
         text-decoration: none;
         color: inherit;
+        outline: none;
     }
 
     &__name {
         font-size: 0.9rem;
         font-weight: 500;
-        word-break: break-all;
+        color: $color-text-dark;
+        line-height: 1.35;
+        word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)